include/odp/api/spec/packet_io_stats.h

```diff
@@ -26,42 +26,23 @@ extern "C" {
  */
 
 /**
- * Packet IO statistics
- *
- * Packet IO statistics counters follow RFCs for Management Information Base
- * (MIB)for use with network management protocols in the Internet community:
- * https://tools.ietf.org/html/rfc3635
- * https://tools.ietf.org/html/rfc2863
- * https://tools.ietf.org/html/rfc2819
+ * Packet IO statistics counters
  */
 typedef struct odp_pktio_stats_t {
-	/**
-	 * The number of octets in valid MAC frames received on this interface,
-	 * including the MAC header and FCS. See ifHCInOctets counter
-	 * description in RFC 3635 for details.
-	 */
+	/** Number of octets in successfully received packets (includes MAC
+	 *  header and FCS). */
 	uint64_t in_octets;
 
-	/**
-	 * The number of packets, delivered by this sub-layer to a higher
-	 * (sub-)layer, which were not addressed to a multicast or broadcast
-	 * address at this sub-layer. See ifHCInUcastPkts in RFC 2863, RFC 3635.
-	 */
+	/** Number of successfully received packets with a unicast destination
+	 *  MAC address. */
```

Comment:
Do we want to distinguish ucast vs bcast/mcast? From the existing counters, it may be not straightforward to calculate the number of bcast/mcast packets.

Reply:
Since we already have a counter for unicast packets, adding counters for multicast and broadcast packets would be quite logical.

Reply:
The number of multicast/broadcast packets can now be calculated from: `in_packets` - `in_ucast_pkts`

We can add the specific counter in a separate PR.